runtime/compiler/control/J9Options.cpp

```diff
@@ -1146,6 +1170,55 @@ static bool JITaaSParseOptionsCommon(char **options, char delimiter, TR::Compila
    return true;
    }
 
+static void JITaaSParseEnvVarCommon(TR::CompilationInfo *compInfo, const struct TR_JTIaaSCommonOptions & optionsSet)
+   {
+   
+   if (!optionsSet.hasAddrOption)
+      {
+      char* address = feGetEnv("TR_JITaaSServerAddress");
+      // j9mem_free_memory(address);
```

Comment:
Adding this line, `j9mem_free_memory`, into the code generates a compile time error that I have no idea what it means. It is related to the macros and probably the private port library? The error only happens in this function, i.e. I can call it from one layer above (where server/client mode is parsed) or from `JITaaSParseOptionsCommon` without having this error.

```
/jre/lib/amd64/compressedrefs/oti/j9port_generated.h:697:55:
error: ‘privatePortLibrary’ was not declared in this scope
#define j9mem_free_memory(param1) OMRPORT_FROM_J9PORT(privatePortLibrary)->mem_free_memory(OMRPORT_FROM_J9PORT(privatePortLibrary),param1)

/jre/lib/amd64/compressedrefs/oti/j9port.h:59:44: 
note: in definition of macro ‘OMRPORT_FROM_J9PORT’
#define OMRPORT_FROM_J9PORT(_j9portLib) (&(_j9portLib)->omrPortLibrary)

/runtime/compiler/control/J9Options.cpp:1179:7: 
note: in expansion of macro ‘j9mem_free_memory’
j9mem_free_memory(address);
```

Reply:
You don't need to free the memory from feGetEnv.